sdk/storage/src/blob/blob/block_with_size_list.rs

```diff
@@ -107,9 +104,10 @@ mod test {
 
         assert!(
             bl.blocks[0].block_list_type
-                == BlobBlockType::Committed(Vec::from(b"base64-encoded-block-id" as &[u8]))
+                == BlobBlockType::new_committed(Bytes::from_static(b"base64-encoded-block-id"))
```

Comment:
There's no need to use the `Bytes::from_static` since `new_commited` takes an `Into<Bytes>`

Reply:
That is what I was hoping, but the compiler doesn't like it. If you define it as a variable above, it works. That was my experience at least.

Reply:
You need to either change the byte string to be a regular `&str` or make it a slice `&b"base64-blah"[..]`